app/config/drupal8-demo/download.sh

```diff
@@ -4,20 +4,29 @@
 
 ###############################################################################
 
-[ -z "$CMS_VERSION" ] && CMS_VERSION=8.x
+[ -z "$CMS_VERSION" ] && CMS_VERSION=8.9.x-dev
 [ -z "$VOL_VERSION" ] && VOL_VERSION='master'
 [ -z "$NG_PRFL_VERSION" ] && NG_PRFL_VERSION='master'
-[ -z "$DISC_VERSION" ] && DISC_VERSION=master
+[ -z "$RULES_VERSION" ] && RULES_VERSION='master'
+[ -z "$DISC_VERSION" ] && DISC_VERSION='master'
 
-MAKEFILE="${TMPDIR}/${SITE_TYPE}/${SITE_NAME}/${SITE_ID}.make"
-cvutil_makeparent "$MAKEFILE"
-cat "$SITE_CONFIG_DIR/drush.make.tmpl" \
-  | sed "s;%%CACHE_DIR%%;${CACHE_DIR};" \
-  | sed "s;%%CIVI_VERSION%%;${CIVI_VERSION};" \
-  | sed "s;%%CMS_VERSION%%;${CMS_VERSION};" \
-  | sed "s;%%DISC_VERSION%%;${DISC_VERSION};" \
-  | sed "s;%%VOL_VERSION%%;${VOL_VERSION};" \
-  | sed "s;%%NG_PRFL_VERSION%%;${NG_PRFL_VERSION};" \
-  > "$MAKEFILE"
+mkdir "$WEB_ROOT"
+composer create-project drupal/recommended-project:"$CMS_VERSION" "$WEB_ROOT" --no-interaction
 
-drush8 -y make --working-copy "$MAKEFILE" "$WEB_ROOT"
+pushd "$WEB_ROOT" >> /dev/null
```

Comment:
This could use the composer install function from civibuild.lib.sh like the d8prj builds do.

Reply:
Same for D9 :-)

Reply:
@MikeyMJCO is that not what is called in L25 there?

Reply:
Yes, ignore me :-)